src/libraries/ExcelConverter/Utils.cs

```diff
@@ -317,9 +332,181 @@ public static String QuoteWrap(String input)
+        {
+            Match match = internalTableRefRegex.Match(formula);
+            output = formula;
+
+            if (!match.Success) return false;
+
+            String tableName = match.Groups[1].Value;
+            String operationAxis = match.Groups[2].Value; 
+            String column1 = match.Groups[3].Value;
+            String column2 = match.Groups[4].Value;
+
+            ExcelParser.ParsedTable tableObj = null;
+
+            // If this isn't a valid table name, just return the unmodified formula
+            if (!Converter.tableMap.TryGetValue(tableName, out tableObj)) return false;
+
+            ExcelParser.ParsedTableColumn column1Obj = null;
+            ExcelParser.ParsedTableColumn column2Obj = null;
+
+            tableObj.ColumnMap.TryGetValue(column1, out column1Obj);
+            tableObj.ColumnMap.TryGetValue(column2, out column2Obj);
+
+            String constructedRange = "";
+            Match cellMatch = currCellRegex.Match(cell.CellId);
+            int currRowNum = int.Parse(cellMatch.Groups[2].Value); // row the current cell is on
+            // If match found for column1 AND column2
+            if (column1Obj != null && column2Obj != null)
+            {
+                // With two columns being referenced, range start is column 1's cell that lies on the current row
+                // by the same reasoning, range end is column 2's cell that lies on the same row
+                String column1SpanStart = column1Obj.columnSpan.startChar + currRowNum.ToString();
+                String column2SpanEnd = column2Obj.columnSpan.endChar + currRowNum.ToString();
+                constructedRange = column1SpanStart + ":" + column2SpanEnd;
+            }
+            else // If match only found for column1
+            {
+                // In this case, we actually only have a singular cell which is column 1's cell that lies on the current row
+                constructedRange = column1Obj.columnSpan.startChar.ToString() + currRowNum;
+            }
+
+            output = Regex.Replace(formula, internalTableRefRegex.ToString(), constructedRange);
+            return true;
+        }
+
+        // Corrects Excel function names to the proper PowerFx equivalent name
+        // Eg. VLOOKUP -> LOOKUP
+        public static String AlignFunctionName(String functionName)
+        { 
+            if (functionName == null) return null;
+
+            // If we have a match (there's a name switch that needs to be made)
+            // Then we return the proper name
+            String output = "";
+            if (functionNameMap.TryGetValue(functionName, out output)) return output;
+
+            return functionName;
+        }
+
+
+        // Takes in a parsed table object and builds an output table
+        // Example output: Table({Column1=B1, Column2=B2, Column3=B3, Column4=B4}, {Column1=C1, Column2=C2, Column3=C3, Column4=C4})
+        public static String BuildTableOutput(ExcelParser.ParsedTable tableInput)
```

Comment:
I think this function would be a good ToString override for the ParsedTable object

Reply:
will resolve later during refactoring phase